src/ReactotronConfig.js

```diff
@@ -5,16 +5,19 @@ import { NativeModules } from 'react-native';
 import { reactotronRedux } from 'reactotron-redux';
 
 // uncomment the following lines to test on a device
-// let scriptHostname;
-// if (__DEV__) {
-//   const scriptURL = NativeModules.SourceCode.scriptURL;
-//   scriptHostname = scriptURL.split('://')[1].split(':')[0];
-// }
 
-Reactotron.configure(/*{ host: scriptHostname }*/) // controls connection & communication settings
+if (__DEV__) {
+  const scriptURL = NativeModules.SourceCode.scriptURL;
+  const scriptHostname = scriptURL.split('://')[1].split(':')[0]; //To retrieve scriptURL from device
+
+  console.logs = (text) => Reactotron.log(text,true);
+
+  //Comment out next lines to disable reactotron connection
+  Reactotron.configure({ host: scriptHostname, name: "Zulip Mobile" }) // controls connection & communication settings
```

Comment:
Surrounded whole code within `__Dev__` so that there is no harm in production environment.